proto/v1/instance_service.proto

```diff
@@ -232,6 +241,23 @@ message SyncInstanceRequest {
   bool enable_full_sync = 2;
 }
 
+message ListInstanceDatabaseRequest {
+  // The name of the instance.
+  // Format: instances/{instance}
+  string name = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (google.api.resource_reference) = {type: "bytebase.com/Instance"}
+  ];
+
+  // The target instance. We need to set this field if the target instance is not created yet.
+  optional Instance instance = 2 [(google.api.field_behavior) = REQUIRED];
```

Comment:
Is this required? Suppose we haven't created instance yet and we need to pass in the connection info. This is very similar to "Test Connection".

Reply:
It's required during the creation, but optional for existed instance